cinema/migrations/0003_alter_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-16 16:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0002_movie_image'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(blank=True, null=True, upload_to='uploads/movies/'),

```

Comment:
The `upload_to` parameter should use a callable that formats the path with `slugify` and `uuid`, as required by the task. Please update this to meet the task's requirements.